doc/conf.py

```diff
@@ -65,8 +65,8 @@
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
-from pkg_resources import get_distribution
```

Comment:
`pkg_resources` is from setuptools[^1], let's use something in the stdlib instead

[^1]: might be `distutils`, but better to avoid it anyway